Dockerfile-default-cpu

```diff
@@ -22,6 +22,7 @@ ARG HOROVOD_WITH_PYTORCH=1
 ARG HOROVOD_WITHOUT_MXNET=1
 ARG HOROVOD_WITHOUT_MPI=1
 ARG HOROVOD_CPU_OPERATIONS=GLOO
+RUN pip install cmake==3.22.4
```

Comment:
does it require this new version? if so, 
1. should we skip `apt-get install`ing cmake in `Dockerfile-base-*`
2. can we install the same cmake version in both `-cpu` and `-gpu` so they match

Reply:
👍